rootfs_overlay/etc/boardid.config

```diff
@@ -0,0 +1,7 @@
+# boardid.config
+
+# Read the serial number from the U-boot environment block.
+-b uboot_env -u nerves_serial_number
+
+# Use the MAC address of the first Ethernet port.
+-b macaddr -n 4
```

Comment:
It feels a little strange using the MAC address on a VM. Is the Ethernet port bridged or is there a virtual Ethernet port? Either way seems like the MAC address would be unreliable. This may just be something we fix later.